projects/packages/videopress/src/client/lib/token-bridge/index.ts

```diff
@@ -88,7 +91,7 @@ export async function tokenBridgeHandler(
 			guid,
 			requestId,
 		},
-		'*'
+		{ targetOrigin: '*' }
```

Comment:
Reason:
![2023-02-06_17-46-49](https://user-images.githubusercontent.com/8486249/217081537-59989aca-677b-4adf-9994-861b2e2b7022.png)

`'*'` does work in the first signature, but TS shows an error anyway:
![2023-02-06_17-47-01](https://user-images.githubusercontent.com/8486249/217081534-63704731-fdec-4cb4-b1bd-9d95c756bbdf.png)

![2023-02-06_17-47-10](https://user-images.githubusercontent.com/8486249/217081529-752c0d78-4619-4852-b096-a087679ca0a4.png)

Reply:
So the change is to use another signature for the same function, using the `options` object instead of the straight string `targetOrigin`?

Reply:
Yes. It should not change the functionality, just get rid of the "error".